### PR TITLE
Allow dependency checker to run on more subprojects

### PR DESCRIPTION
**Pull request by lptr**
### Reviewing cheatsheet

Before merging the PR, comments starting with 
- ❌ ❓**must** be fixed
- 🤔 💅 **should** be fixed
- 💭 **may** be fixed
- 🎉 celebrate happy things

### CODE REVIEW

**Comment by lptr** [comment]
@bot-gradle test this please.

File: gradle.properties [hunk 1]
```diff
@@ -27,7 +27,12 @@ systemProp.dependency.analysis.test.analysis=false
 
 # List of project that still have dependency analysis warnings
 unmigratedProjects=\
-  ide,ide-native,installation-beacon,kotlin-dsl,kotlin-dsl-plugins,\
```
Comment (mlopatkin) [inline gradle.properties:30]: 🎉 nice touch!

**Review by mlopatkin — APPROVED** [review]
There's also https://github.com/gradle/gradle/pull/29229, that aims at the same thing, but I'd prefer your approach.

**Review by bamboo — APPROVED** [review]
👍

**Pull request merged**